15/slim/debian/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM debian:buster-slim
+
+ARG version=15.0.0.36-1
+# In addition to installing the Amazon corretto, we also install
+# fontconfig. The folks who manage the docker hub's
+# official image library have found that font management
+# is a common usecase, and painpoint, and have
+# recommended that Java images include font support.
+#
+# See:
+#  https://github.com/docker-library/official-images/blob/master/test/tests/java-uimanager-font/container.java
+#
+# Slim:
+#   JLink is used (retaining all modules) to create a slimmer version of the JDK excluding man-pages, header files and debugging symbols - saving ~113MB.
+RUN set -eux \
```

Comment:
I believe your boolean ANDs obviates `-e` usage. In my testing, the build fails with or without `-e` when any command in the chain fails.

Reply:
Done. I also tested a failing command in the chain and saw the build fail.